source/_integrations/roku.markdown

```diff
@@ -152,6 +159,54 @@ actions:
       media_content_type: channel
 ```
 
+## Play on Roku
+
+The `media_player.play_media` service may be used to send media URLs (primarily videos) for direct playback on your device.
+
+This feature previously made use of the built-in PlayOnRoku application that was available prior to Roku OS 11.5. Since Home Assistant v2024.10.3, you can set a third-party application that supports the PlayOnRoku API via the `Play Media Roku Application ID` option.
```

Comment:
```suggestion
This feature previously made use of the built-in PlayOnRoku application that was available before Roku OS 11.5. Since Home Assistant v2024.10.3, you can set a third-party application that supports the PlayOnRoku API via the `Play Media Roku Application ID` option.
```